ravenframework/SupervisedLearning/ScikitLearn/GaussianProcess/GaussianProcessRegressor.py

```diff
@@ -129,26 +129,25 @@ class cls.
-                                                 default=1e-10))
+                                                           values form a positive definite matrix. It can also be interpreted as the variance of additional Gaussian measurement noise on the training observations. \default{1e-10}""",default=1e-10))
     specs.addSub(InputData.parameterInputFactory("n_restarts_optimizer", contentType=InputTypes.IntegerType,
                                                  descr=r"""The number of restarts of the optimizer for finding the kernel's parameters which maximize the log-marginal likelihood. The first run of the optimizer is performed
                                                            from the kernel's initial parameters, the remaining ones (if any) from thetas sampled log-uniform randomly from the space of allowed theta-values. If greater than
-                                                           0, all bounds must be finite. """, default=0))
+                                                           0, all bounds must be finite. \default{0}""", default=0))
     specs.addSub(InputData.parameterInputFactory("normalize_y", contentType=InputTypes.BoolType,
                                                  descr=r"""Whether the target values y are normalized, the mean and variance of the target values are set equal to 0 and 1 respectively. This is recommended for cases where zero-mean,
-                                                           unit-variance priors are used.""", default=False))
+                                                           unit-variance priors are used. \default{False}""",default=False ))
     specs.addSub(InputData.parameterInputFactory("random_state", contentType=InputTypes.IntegerType,
-                                              descr=r"""Seed for the internal random number generator""", default=None))
+                                              descr=r"""Seed for the internal random number generator. \default{None}""",default=None))
     specs.addSub(InputData.parameterInputFactory("optimizer", contentType=InputTypes.makeEnumType("optimizer", "optimizerType",['fmin_l_bfgs_b']),
-                                                 descr=r"""Per default, the 'L-BGFS-B' algorithm from
-                                                 scipy.optimize.minimize is used. If None is passed, the kernel’s
-                                                 parameters are kept fixed. """, default='L-BGFS-B'))
+                                                 descr=r"""Per default, the 'L-BFGS-b' algorithm from
+                                                 scipy.optimize.minimize is used. If None is passed, the kernel\’s
+                                                 parameters are kept fixed. \default{'fmin_l_bfgs_b'}""",default='fmin_l_bfgs_b'))
```

Comment:
I think we should avoid to add \default directly in the 'descr'. In this case, we need to update all our roms. I would suggest that we could be able to use the default option to automatic generate \default tex msg, in this case, all our roms will be get updated instead of manual updates.